plugins/actix-web/src/lib.rs

```diff
@@ -248,6 +251,24 @@ where
         self
     }
 
+    /// Mounts the specification for all operations and definitions
+    /// recorded by the wrapper and serves them in the given path
+    /// as a JSON with Swagger UI.
+    #[cfg(feature = "swagger-ui")]
+    pub fn with_swagger_ui_at(mut self, path: &str) -> Self {
+        self.inner = self.inner.take().map(|a| {
+            a.service(
+                actix_web::web::resource(path)
+                    .route(actix_web::web::get().to(SpecHandler(self.spec.clone()))),
+            )
+            .service(
+                actix_web::web::resource(path.to_owned() + "/swagger-ui.html")
```

Comment:
This doesn't feel right. We need to think of a better way. I understand that the swagger UI needs access to the JSON spec, but we shouldn't mount the UI at some non-configurable path (what if users wanted this to be at `/api/swagger`?).

I propose to have a `path: Option<String>` field in the `App` wrapper type. We set this path to the JSON spec path (obtained when the user calls `with_json_spec_at` and we mount the UI handler only if this path is set, so if it's not set, we'll get a 404 from backend.

Reply:
Ok, I will add it

Reply:
I have made the changes you requested.